packages/menu/src/Menu.story.tsx

```diff
@@ -28,7 +28,7 @@ export default StoryMeta({
     },
   },
   args: {
-    open: true,
+    open: undefined,
```

Comment:
false?

Reply:
It should be undefined so it's uncontrolled by default